tests/framework/utils/k8s_helper.go

```diff
@@ -568,17 +568,33 @@ func (k8sh *K8sHelper) GetEventsFromNamespace(namespace, testName, platformName
 	if events == "" {
 		return
 	}
-	file.WriteString(events)
+	_, err = file.WriteString(events)
+	if err != nil {
+		return
+	}
+
 }
 
 func (k8sh *K8sHelper) appendPodDescribe(file *os.File, namespace, name string) {
 	description := k8sh.getPodDescribe(namespace, name)
 	if description == "" {
 		return
 	}
-	writeHeader(file, fmt.Sprintf("Pod: %s\n", name))
-	file.WriteString(description)
-	file.WriteString("\n")
+	err := writeHeader(file, fmt.Sprintf("Pod: %s\n", name))
```

Comment:
Let's actually add nolint for all the errors in this test file. We don't get much value for these helpers to check for err.

Reply:
Done.